docs/hugo/content/guide/authentication/credential-scope.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Credential Scope
+linktitle: Credential Scope
+---
+
+When presented with multiple credential choices, the operator chooses the most specific one:
+resource scope takes precedence over namespace scope which takes precedence over global scope.
+
+## Global scope
```

Comment:
Also mention about this being optional somewhere?

Reply:
Something like : `This Service Principal is used as a default global credential and is optional. If you **do not** wish to use a default global credential, you must still provide the secret with empty values for the below keys and follow the instructions in to specify per-namespace or per-resource credentials`

Reply:
I put this at the bottom rather than the top:

> **Note**: The `aso-controller-settings` secret must exist, but the actual global credential portion of the secret
> **is optional**. If you do not wish to use a global credential, you must still provide the secret with empty values
> for the `AZURE_SUBSCRIPTION_ID`, and `AZURE_CLIENT_ID`, and `AZURE_TENANT_ID` strings.